test/Sema/struct_equatable_hashable_access.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -print-ast %s 2>&1 | %FileCheck %s
+// RUN: %target-swift-frontend -print-ast-decl %s 2>&1 | %FileCheck %s
```

Comment:
It might be useful to add tests for the synthesized _bodies_ for `Equatable`/`Hashable`/etc. But this existing test just wants to check the access level. Using the new `-print-ast-decl` makes this the least disruptive.